src/buffers/TranscodedBuffer.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+/*
+*      Copyright (C) 2020
+*
+*  This Program is free software; you can redistribute it and/or modify
+*  it under the terms of the GNU General Public License as published by
+*  the Free Software Foundation; either version 2, or (at your option)
+*  any later version.
+*
+*  This Program is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with XBMC; see the file COPYING.  If not, write to
+*  the Free Software Foundation, 51 Franklin Street, Fifth Floor, Boston,
+*  MA 02110-1301  USA
+*  http://www.gnu.org/copyleft/gpl.html
+*
+*/
+
+#include  "../BackendRequest.h"
+#include "DummyBuffer.h"
+#include <sstream>
+using namespace ADDON;
```

Comment:
`using namespace` in headers is really very bad coding style

Reply:
The buffer class design originated with another user in 2018 https://github.com/kodi-pvr/pvr.nextpvr/commit/3b5b0a89e6276b67e2eca2837e910b8daec80611#diff-fcaaa45c893f266e0a2dac97e871ae8d but I can remove it from this class and modify other classes as I touch them.

Reply:
Please do. We should not replicate old bugs when we copy of code.

Reply:
I removed this from the classes I touched in the PR